modules/ROOT/pages/cap-policy.adoc

```diff
@@ -3,39 +3,42 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The HTTP Limits policy prevents an attacker from sending large messages that consume all your bandwidth. +
-This policy checks TCP protocol message sizes and headers. This policy does not check content.
+Anypoint Security uses the HTTP Limits policy to prevent an attacker from sending large messages that can consume bandwidth and result performance degradation or system outage. This policy applies to all the APIs behind your Edge inbound endpoint. You can apply API Gateway policies to each API to enforce other API-specific policies, such as throttling and JSON threat protection.
```

Comment:
This looks a bit dicey: "large messages that can consume bandwidth and result performance degradation or system outage". Maybe something like: "large messages that can consume bandwidth which can cause performance degradation or denial of service."